client/src/pages/Login.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react";
+import {Link} from 'react-router-dom';
+import {useStyles} from '../themes/loginSignup';
+import { Button, Typography, TextField } from "@material-ui/core";
+
+function Login() {
+  const classes = useStyles();
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(email, password);
+    // API verification steps ...
+  };
+
+  return (
+    <div className={classes.main}>
+      <div className={classes.contentBox}>
+          <img className={classes.img} src="/images/sign-up.png" alt="women-computing" />
+        </div>
+        <div className={classes.contentBox}>
+          <div className={classes.upperDetails}>
+            <form className={classes.form}>
+              <Typography 
+                className="typography" 
+                variant="h1"
+              >
+                Welcome back!
+              </Typography>
+              <TextField 
+                className={classes.textField}
+                variant="outlined"
+                label="Enter email"
+                value={email}
+                onChange={(e) => setEmail(e.target.value)}
+              />
+              <TextField 
+                className={classes.textField}
+                variant="outlined"
+                label="Password"
```

Comment:
For this and the one in Signup, can you add `type="password"` so that the password will be hidden?

Reply:
Thanks Dan, I don't know how I missed that. It must have happened in the change from HTML to material-ui. I have just updated the Signup and Login components.